src/WebJobs.Script/Config/TransmissionStatusHandler.cs

```diff
@@ -44,6 +44,10 @@ internal static string FormattedLog(object sender, TransmissionStatusEventArgs a
                             Count = n.Count()
                         });
 
+            // Log all the unique ikeys in the transmission, avoid storing the complete key.
+            IEnumerable<string> iKeys = transmission.TelemetryItems.GroupBy(n => n.Context.InstrumentationKey)
+                                        .Select(n => n.Key.Length > 24 ? n.Key.Substring(0, 24) : n.Key);
```

Comment:
Where did 24 come from? Can we get confirmation that this is enough of a trim?

Also... maybe put something to indicate that the rest was redacted? I think there are examples elsewhere where we (or antares?) does this today? Like how the Sanitizer uses `12345-[Hidden Credential]`?

Reply:
@4 gives us good enough details about the IKey. I did check this with the Todd (AI EEE) and he is good with this.
I have added ************ to make it obvious that we have sanitized it.

Diagnostic source 'Microsoft.Azure.Functions.Host.ApplicationInsights' emitted event 'TransmissionStatus': {
  "statusCode": 400,
  "statusDescription": "Bad Request",
  "id": "5kmhwqKFP8Y=",
  "responseTimeInMs": 1,
  "endpointAddress": "https://eastus-8.in.applicationinsights.azure.com/v2/track",
  "items": [
    {
      "count": 10,
      "type": "AppTraces"
    },
    {
      "count": 5,
      "type": "AppMetrics"
    }
  ],
  "iKeys": [
    "9e45c058-e012-4387-a6a4-************"
  ]
}